api/config.py

```diff
@@ -74,6 +75,13 @@ class Config(object):
     DISABLE_NAMEREQUEST_NRO_UPDATES = int(os.getenv('DISABLE_NAMEREQUEST_NRO_UPDATES', 0))
     DISABLE_NAMEREQUEST_SOLR_UPDATES = int(os.getenv('DISABLE_NAMEREQUEST_SOLR_UPDATES', 0))
 
+    # NATS
+    NATS_SERVERS = os.getenv('NATS_SERVERS')
+    NATS_CLIENT_NAME = os.getenv('NATS_CLIENT_NAME')
+    NATS_CLUSTER_ID = os.getenv('NATS_CLUSTER_ID')
+    NATS_QUEUE = os.getenv('NATS_QUEUE')
+    NATS_NR_STATE_SUBJECT = os.getenv('NATS_NR_STATE_SUBJECT', 'namex.event')
```

Comment:
Will you be including these into openshift files as part of this PR?

Reply:
Should I do it? Sorry, I don't know how it work in openshift.

Reply:
@thorwolpert should I add this conf to openshift? Is there any reference how to do that?

Reply:
I talked to @amichard and he's adding them to onepass.